public/js/graph/react_graph.js

```diff
@@ -139,8 +139,13 @@ var Graph = React.createClass({
 
     componentDidMount: function () {
         this.getGraph();
-        document.body.addEventListener('keydown', this.onKeyDown);
-        document.body.addEventListener('wheel', this.onWheel);
+        document.getElementById('react-graph').addEventListener('keydown', this.onKeyDown);
+        document.getElementById('react-graph').addEventListener('wheel', this.onWheel);
+    },
+
+    componentDidUnmount: function () {
```

Comment:
Thanks! That was a silly mistake seeing as componentDidUnmount doesn't even exist. Giving that document a read, I think I now better understand why exactly that is.